research-hub-web/.eslintrc.js

```diff
@@ -0,0 +1,54 @@
+module.exports = {
+  root: true,
+  ignorePatterns: [
+    "projects/**/*",
```

Comment:
projects?

Reply:
Interesting, didn't catch that. The migration tool that I used must've put it in there. I'll remove it.